### DISCUSSION TITLE
Error shown in Solodit platform whenever I use Report Tags

### DISCUSSION THREAD

**Discussion by ADK0010** [solodit/feedback#5] [Q&A] [answered]
I think there is some problem with 
Solodit platform. It is showing "SyntaxError: Unexpected token '<', " <!doctype "... is not valid JSON" when I use the report tags as shown in this tweet
https://twitter.com/AdityaK0010/status/1666099635730120707?s=20

**Comment by wang1252** [comment, marked as answer]
We have resolved the bug. Thank you for report..
